index.html

```diff
@@ -1 +1,2 @@
 <h1>Hello World</h1>
+<p>steven</p>
```

Comment:
The first letter needs to be capitalized

Reply:
I changed the capital letter for the name.